draft-ietf-mls-architecture.md

```diff
@@ -1818,8 +1825,8 @@ service matter, not via technology.
 
 Because the DS is responsible for providing the initial keying material to
 clients, it can provide stale keys. This does not inherently lead to compromise
-of the message stream, but does allow it to attack forward security to a limited
-extent. This threat can be mitigated by having initial keys expire.
+of the message stream, but does allow it to attack post-compromise security to
+a limited extent. This threat can be mitigated by having initial keys expire.
```

Comment:
I don't think that we need the text above, but this fix fs => pcs is a good catch

Reply:
Maybe the text above is not in the correct place, but I feel like something needs to be said. The only place where the document says KeyPackage must expire (through the `lifetime` field) is here, where it only talks about the DS replaying old KeyPackages. But even with a honest DS, KeyPackages must expire for PCS.

Reply:
The enforcement of the lifetime field is specified here: https://www.rfc-editor.org/rfc/rfc9420.html#section-7.3-4.5.1
So the text above seems duplicate, no?

Reply:
You are right, the text above is duplicate with the mls-protocol rfc.

I think the text should be something saying that short lifetimes is better for PCS (and longer lifetimes is better for availability or something). Right now the mls-architecture document is not really talking about the implications of lifetimes, or is too vague about it (e.g. only talking about evil DS replaying old KeyPackages).

I don't know where such a text should belong though, I am not highly familiar with the mls-architecture rfc. What do you think about it?

Reply:
That concern would seem to already be addressed by the paragraph we're commenting on: "the DS can provide stale keys; the solution is to set an expiration"

Also here: https://www.rfc-editor.org/rfc/rfc9420.html#section-16.6-5

Reply:
Ok, this paragraph is exactly about what I wanted to mention in this document. I removed the text above.